<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesCamera</class>
 <widget class="QWidget" name="PropertiesCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMode">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>7</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Perspective</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Orthographic</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelFieldOfView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>98</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Field Of View</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderFieldOfView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>98</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFieldOfView">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>97</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelClipPlanes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>128</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Clip Planes</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNearPlane">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>127</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Near</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxNearPlane">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>127</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>10.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelFarPlane">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>153</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Far</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxFarPlane">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>152</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
   <property name="value">
    <double>1000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelViewportRatio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>188</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Viewport Ratio</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelViewportScaleWidth">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>191</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxViewportWidth">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>186</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.250000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelViewportScaleHeight">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>216</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxViewportHeight">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>213</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.250000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxClear">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Color</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sky</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Depth Only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nothing</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonColor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonColorEyeDropper">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>70</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../gameplay-editor.qrc">
     <normaloff>:/res/images/eyedropper.png</normaloff>:/res/images/eyedropper.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>98</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxSize">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>97</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <zorder>labelSize</zorder>
  <zorder>doubleSpinBoxSize</zorder>
  <zorder>labelMode</zorder>
  <zorder>comboBoxMode</zorder>
  <zorder>labelFieldOfView</zorder>
  <zorder>horizontalSliderFieldOfView</zorder>
  <zorder>spinBoxFieldOfView</zorder>
  <zorder>labelClipPlanes</zorder>
  <zorder>labelNearPlane</zorder>
  <zorder>doubleSpinBoxNearPlane</zorder>
  <zorder>labelFarPlane</zorder>
  <zorder>doubleSpinBoxFarPlane</zorder>
  <zorder>labelViewportRatio</zorder>
  <zorder>labelViewportScaleWidth</zorder>
  <zorder>doubleSpinBoxViewportWidth</zorder>
  <zorder>labelViewportScaleHeight</zorder>
  <zorder>doubleSpinBoxViewportHeight</zorder>
  <zorder>label</zorder>
  <zorder>comboBoxClear</zorder>
  <zorder>labelColor</zorder>
  <zorder>toolButtonColor</zorder>
  <zorder>pushButtonColorEyeDropper</zorder>
 </widget>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
</ui>
